runtime/jit_vm/ctsupport.cpp

```diff
@@ -202,6 +202,7 @@ jitParseSignature (const J9UTF8 *signature, U_8 *paramBuffer, UDATA *paramElemen
 			state = returnValue;
 		} else {
 			switch (*sigChar) {
+			case 'Q':
 			case 'L': next = J9_NATIVE_TYPE_OBJECT; break;
```

Comment:
```suggestion
			case 'Q': /* FALLTHROUGH */
			case 'L': next = J9_NATIVE_TYPE_OBJECT; break;
```

Reply:
It's a minor thing but we try to label all fall throughs explicitly so that future refactoring doesn't mess up the logic